WooCommerce/Classes/Authentication/Store Creation/Profiler/Category/StoreCreationCategoryQuestionView.swift

```diff
@@ -0,0 +1,65 @@
+import SwiftUI
+
+/// Hosting controller that wraps the `StoreCreationCategoryQuestionView`.
+final class StoreCreationCategoryQuestionHostingController: UIHostingController<StoreCreationCategoryQuestionView> {
+    init(viewModel: StoreCreationCategoryQuestionViewModel) {
+        super.init(rootView: StoreCreationCategoryQuestionView(viewModel: viewModel))
+    }
+
+    @available(*, unavailable)
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureNavigationBarAppearance()
+    }
+
+    /// Shows a transparent navigation bar without a bottom border and with a close button to dismiss.
+    func configureNavigationBarAppearance() {
+        let appearance = UINavigationBarAppearance()
+        appearance.configureWithTransparentBackground()
+        appearance.backgroundColor = .systemBackground
+
+        navigationItem.standardAppearance = appearance
+        navigationItem.scrollEdgeAppearance = appearance
+        navigationItem.compactAppearance = appearance
+    }
```

Comment:
I see that we're using this style a lot so I added an extension for this [here](https://github.com/woocommerce/woocommerce-ios/blob/69be467b056c9ef4bb7cd3c5c2220970880fc4db/WooCommerce/Classes/Extensions/UIViewController%2BHelpers.swift#L38). Maybe we should use it instead?

Reply:
Yup thanks for creating that extension, updated in https://github.com/woocommerce/woocommerce-ios/pull/8379/commits/87be413bba013e4b19549c0f75fdbe798ab77ea3